genesyscloud/outbound_contact_list_contact/genesyscloud_outbound_contact_list_contact_proxy.go

```diff
@@ -0,0 +1,74 @@
+package outbound_contact_list_contact
+
+import (
+	"context"
+	"github.com/mypurecloud/platform-client-sdk-go/v130/platformclientv2"
+)
+
+var internalProxy *contactProxy
+
+type createContactFunc func(ctx context.Context, p *contactProxy, contactListId string, contact platformclientv2.Writabledialercontact, priority, clearSystemData, doNotQueue bool) ([]platformclientv2.Dialercontact, *platformclientv2.APIResponse, error)
+type readContactByIdFunc func(ctx context.Context, p *contactProxy, contactListId, contactId string) (*platformclientv2.Dialercontact, *platformclientv2.APIResponse, error)
+type updateContactFunc func(ctx context.Context, p *contactProxy, contactListId string, contactId string, contact platformclientv2.Dialercontact) (*platformclientv2.Dialercontact, *platformclientv2.APIResponse, error)
+type deleteContactFunc func(ctx context.Context, p *contactProxy, contactListId, contactId string) (*platformclientv2.APIResponse, error)
+
+type contactProxy struct {
+	clientConfig        *platformclientv2.Configuration
```

Comment:
What about a cache call for the get/getAll functions on exports.  Any concerns about this?